### PR TITLE
setup lint

### PR DESCRIPTION
**Pull request by alpknx**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	"rules": {
+	  "no-bitwise": 0,
+	  "react/react-in-jsx-scope": "off",
+	  "react-hooks/rules-of-hooks": "error",
+	  "react-hooks/exhaustive-deps": "warn",
+	  "@typescript-eslint/ban-ts-comment": "off",
+	  "@typescript-eslint/explicit-module-boundary-types": "off",
+	  "@typescript-eslint/no-explicit-any": "off"
+	}
+ }
```
Comment (klikkn) [inline .eslintrc.json:36]: add an empty row pls here and below

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+	"singleQuote": true,
+	"jsxBracketSameLine": false,
+	"useTabs": false,
+	"eslintIntegration": false,
+	"tslintIntegration": true,
```
Comment (klikkn) [inline .prettierrc:6]: you do not need tslint, it was deprecated

File: package.json [hunk 3]
```diff
@@ -20,7 +20,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+	 "lint": "eslint \"**/*.{ts,tsx,js,jsx}\"",
```
Comment (klikkn) [inline package.json:24]: format this row

**Review by klikkn — CHANGES_REQUESTED** [review]

**Review by klikkn — APPROVED** [review]

File: src/api/api.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+export const API_URL = "https://instagram-api-alex-p.herokuapp.com/api"
+
+const $api  = axios.create({
+	withCredentials: true, 
+	baseURL: API_URL
+})
+
+$api.interceptors.request.use(config => {
+	config.headers.Authorization = `Bearer ${localStorage.getItem('token')}`
```
Comment (alpknx) [inline src/api/api.ts:12]: why config.headers as undefinded object?

File: src/pages/Login/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import { useActions } from '../../hooks/useActions';
+
+const Login:FC = () => {
+   const [username, setUsername] = useState('')
+   const [password, setPassword] = useState('')
+   const {login} = useActions()
+	const submit = () => {
+		login(username, password)
+	}
+
```
Comment (alpknx) [inline src/pages/Login/Login.tsx:11]: need an advice about how to mix usesState and element

**Pull request closed without merging**